### DISCUSSION TITLE
Prefetching Should Be Default

### DISCUSSION THREAD

**Discussion by ramonmalcolm10** [inertiajs/inertia#2043] [Ideas]
Any reason why prefetching is not the default option, or is it possible to add a configuration to allow person to enable/disable globally?

**Comment by joelstein** [comment]
I would love this! I think a default prefetch option for GET requests would be awesome. (So I don't forget to add them to every link.)

And then maybe we could set something like `prefetch={false}` for any `<Link>` elements we _don't_ want to prefetch.
